package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -25,6 +33,14 @@ export interface SpotlightTourProviderProps {
    * argument.
    */
   children: React.ReactNode | ChildFn<SpotlightTour>;
+  /**
+   * Specifies {@link FloatingProps} in order to configure Floating UI
+   * in all tour steps layout.
+   *
+   * @default middlewares: [flip(), offset(4), shift()]
+   * @default placement: "bottom"
```

Comment:
This is great! Can you also add this info somewhere in the readme? I think that will be really useful for users 🙂

Reply:
I added this information in the readme and updated the examples since they were using the old align to and positioning